specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/applicationGateway.json

```diff
@@ -451,6 +451,96 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableServerVariables": {
```

Comment:
these resource type names are pretty long, can they be condensed without losing meaning?

Reply:
Actually NRP dev code for these API's has been checked in with these resource type names. So i think for now we can keep these names so as to not to break the existing changes.

Reply:
ok